misc/acrn-config/library/scenario_cfg_lib.py

```diff
@@ -178,8 +185,11 @@ def get_ttys_info(board_info):
         if not ttys_line:
             break
 
-        #ttys_dev = " ".join(ttys_line.strip().split()[:-2])
         ttys_dev = ttys_line.split()[0].split(':')[1]
+        ttysn = ttys_dev.split('/')[-1]
+        # currently SOS can only emulate legacy serial port
```

Comment:
Is the comment accurate? HV emluates the legacy serial port or SOS?

Reply:
Serial console port in sos vm should be consistent with hv.

Reply:
I  will change it to "# currently SOS console can only support legacy serial port"